src/components/stats/SoloRefactor.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+
+import { Query } from 'react-apollo';
+import { gql } from 'apollo-boost';
+import client from '../../utils/GitHubGQL';
+
+class SoloRefactor extends Component {
+  constructor(props) {
+    super();
+    this.state = {
+      startDate: new Date(props.startDate).toISOString(),
+      endDate: `${props.endDate}T23:59:59.999Z`,
+      username: props.username,
+      repository: props.repository,
+      winner: '',
+      highScore: 0,
+    };
+  }
+
+  componentDidMount() {
+    const { username } = this.state;
+    const { repository } = this.state;
+    const { startDate } = this.state;
+    const { endDate } = this.state;
+
+    client.query({
+      query: gql`
+      {
+        repository(owner: "${username}", name: "${repository}") {
+          defaultBranchRef {
+            target {
+              ... on Commit {
+                history(since: "${startDate}" until: "${endDate}") {
```

Comment:
String interpolation not needed (remove brackets dollar and quotes

Reply:
will test more, i am pretty sure gql needs this specific formatting for its query